cmd/server/assets/realmkeys.html

```diff
@@ -0,0 +1,262 @@
+
+            {{if $realm.UseRealmCertificateKey}}
+            <div class="form-group row">
+                <label for="name" class="col-sm-3">Key ID (kid):</label>
+                <div class="col-sm-9">
+                  <input type="text" id="certKeyID" class="form-control" value="{{.activeRealmKey}}" readonly/>
+                </div>
+            </div>
+            <div class="form-group row">
+                <label for="name" class="col-sm-3">Public key:</label>
+                <div class="col-sm-9">
+                {{if .activePublicKey}}
+                <div class="alert alert-secondary" role="alert">
+                <pre>{{.activePublicKey}}</pre>
+                </div>
+                {{else}}
+                <div class="alert alert-danger" role="alert">error loading public key value</div>
+                {{end}}
+                </div>
+            </div>
+            {{end}}
+
+            <div class="form-group row">
+                <div class="col-sm-12">
+                  <button type="submit" class="btn btn-primary btn-block">Save realm certificate settings</button>
+                </div>
+              </div>
+          </form>
+        
+        {{else}}
+        <div class="alert alert-danger" role="alert">
+            This server is not configured to support per-realm certificate
+            signing keys. Please ask your operator about enablign this
+            functionality.
+        </div>
+        {{end}}
+        </div>
+    </div>
+
+    {{if .supportsPerRealmSigning}}
+    <div class="card mb-3">
+        <div class="card-header">
+          Realm public keys
+        </div>
+        <div class="card-body">
+            <form method="POST" action="/realm/keys/create">
+                {{ .csrfField }}
+                <div class="form-group row">
+                  <div class="col-sm-12">
+                    <small class="form-text text-muted">
+                    Creating a new key will not immediatly make that key active. Before starting to use a new
+                    signing key version, you must communicate the new key ID to your exposure notifications
+                    key server operator.
+                    </small>
+                    <button type="submit" class="btn btn-primary btn-block">Create new signing key version</button>
+                  </div>
+                </div>
+            </form>
+
+            {{if not .realmKeys}}
+            <div class="alert alert-warning" role="alert">No keys have been created for {{$realm.Name}}.</div>
+            {{else}}
+
+            <div class="table-responsive">
+                <table class="table table-bordered table-striped">
+                  <thead>
+                    <tr>
+                      <th scope="col" width="30">KeyID</th>
+                      <th scope="col" width="100">Public Key</th>
+                      <th scope="col" width="30">Active</th>
+                      <th scope="col" width="100">Controls</th>
```

Comment:
One of these columns should have no width defined so it floats. Maybe Public Key?